pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -266,8 +266,8 @@ func getMinRowNum(runs []models.Run) models.RowNum {
 
 // renumberRows will update the runs.row_num field with the correct ordinal
 func (r RunRepository) renumberRows(tx *gorm.DB, startWith models.RowNum) error {
-	if startWith <= models.RowNum(0) {
```

Comment:
Can we add a private unit test in the `repositories` package which will test this function?

Reply:
I have been working on this, and I was using a mock for the `tx *gorm.DB`. Though I can test for the function to fail (`startWith <= models.RowNum(0)`), to test for the function to pass, there is an execution that also takes arguments (line 279). For the mock, I always write and expect the execution, ie

```go
mock.ExpectExec(`LOCK TABLE runs`).
		WillReturnResult(sqlmock.NewResult(0, 0))
```

 but there's no Expect Execution with Arguments I have seen. I will keep looking on this, or is there an alternative way to how we intend to approach this

Reply:
yep, I know that `unit` test could be a bit complicated because of that. Can you also take a look into `integration` test for this particular case? It should be a bit easier I hope.

Reply:
It has been covered in this [commit](https://github.com/G-Research/fasttrackml/pull/195/files#diff-948cf7e6f69d6f4393eeb54b77a7322b533ce6775807b37190e9dd86c567b7b9) to the delete_test integration